operator/tests/central/basic-central/10-assert.yaml

```diff
@@ -1,5 +1,10 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestAssert
+commands:
+- script: |
+    set -eu # shell in CI does not grok -o pipefail
+    # On OpenShift, test that Central auth reader rolebinding exists in kube-system.
+    if kubectl get scc > /dev/null 2>&1; then kubectl get rolebinding rhacs-central-auth-reader-${NAMESPACE} -n kube-system; fi
```

Comment:
Is this testing that the `central-auth-reader-${NS}` exists iif the `SecurityContextConstraint` CR exists ?

Reply:
Yes, exit code 0 from `kubectl get scc` is apparently the idiomatic way of checking whether this is an OpenShift cluster.